feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/signer/multisig/MultisigSigner.kt

```diff
@@ -142,14 +142,18 @@ class MultisigSigner(
         )
 
         val requestBusPayload = MultisigExtrinsicValidationRequestBus.Request(validationPayload)
-        val validationResponse = multisigExtrinsicValidationEventBus.handle(requestBusPayload)
-
-        val validationStatus = validationResponse.validationResult.getOrNull()
-        if (validationStatus !is ValidationStatus.NotValid) return
-
-        multisigSigningPresenter.presentValidationFailure(validationStatus.reason)
-
-        throw SigningCancelledException()
+        multisigExtrinsicValidationEventBus.handle(requestBusPayload)
+            .validationResult
+            .onSuccess {
+                if (it is ValidationStatus.NotValid) {
+                    multisigSigningPresenter.presentValidationFailure(it.reason)
+                    throw SigningCancelledException()
+                }
+            }
+            .onFailure {
+                multisigSigningPresenter.presentValidationException(it)
```

Comment:
I think its enough to just re-throw the exception, there is no need to create a separate presentation method
This will produce more consistent behavior with the rest of the signers